packages/extension/webpack.config.js

```diff
@@ -6,7 +6,6 @@ const TerserPlugin = require('terser-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
-const ExtensionReloader = require('webpack-ext-reloader');
```

Comment:
we can also remove from package.json?

Reply:
I see no harm in keeping it since we are planning on adding it back pretty soon after releasing this anyway

Reply:
Fine either way, add a TODO maybe or note so we don't forget what it is.

Reply:
I added a ticket for this here: https://dailydotdev.atlassian.net/browse/MI-107